resources/aws_ec2.go

```diff
@@ -708,3 +737,42 @@ func (obj *AwsEc2Res) snsPostHandler(w http.ResponseWriter, req *http.Request) {
 		log.Printf("%s: Post: %s", obj, string(post))
 	}
 }
+
+// snsMakeTopic creates a topic on aws sns.
+func (obj *AwsEc2Res) snsMakeTopic() (string, error) {
+	// make topic
+	topicInput := &sns.CreateTopicInput{
+		Name: aws.String(SnsTopicName),
+	}
+	topic, err := obj.snsClient.CreateTopic(topicInput)
+	if err != nil {
+		return "", err
+	}
+	log.Printf("%s: Created SNS Topic", obj)
+
+	var topicArn string
+	if topic.TopicArn != nil {
+		topicArn = *topic.TopicArn
+	}
+	return topicArn, nil
```

Comment:
Is returning an empty string for topicArn valid, or should that be an error scenario? Because the current code _could_ return ("", nil) -- Just FYI.

Reply:
It would error out further down the line, but I can nip it in the bud here with an if ... != ""